drivers/mmc/host/sdhci-of-aspeed.c

```diff
@@ -183,6 +199,13 @@ static int aspeed_sdhci_probe(struct platform_device *pdev)
 		goto err_pltfm_free;
 	}
 
+	dev->cd_inverted = 0;
+	host->mmc_host_ops.get_cd = aspeed_get_cd;
```

Comment:
I'm a little bit confused by the setting callback here.

While I was trying to find a better place I found that your revision of this repo is mismatched to the OpenBMC's one.
As a result, these changes have a lot of merge conflicts.

Reply:
The target is Linux upstream master.